api/helpers/gamestate/get-log.js

```diff
@@ -59,14 +59,14 @@ module.exports = {
         case MoveType.JACK:
           return `${player} stole ${opponent}'s ${targetCardName} with the ${playedCardName}.`;
 
-        case MoveType.UNTARGETED_ONE_OFF:
-          return `${player} played the ${playedCardName} as a one-off to 
+        case MoveType.ONE_OFF: {
+          let log = `${player} played the ${playedCardName} as a one-off to 
```

Comment:
This is bugged and likely could be it's own issue. There seems to be a conflation of whether the cards are card objects vs strings.